portfolio.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./font-awesome-4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <div id="image-container">
+    <img id="ellipses" src="./images/Top-circles.png" alt="circles">
+  </div>
+  <nav>
+    <p class="placeholder">John Doe</p>
+    <button id="menu">
+      <span class="line"></span> 
+      <span class="line"></span>
+      <span class="line"></span>
+    </button>
+  </nav>
+  <header>
+    <h1>Justo Rago</h1>
+    <h2>Software Developer</h2>
+  </header>
+  <section id="support">
+    <h3 id="blurb">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</h3>
```

Comment:
- [ ] This is not a sub-header but a paragraph description, so I would suggest that you use a `<p>` instead of a `<h3>` tag for semantic and accessibility reasons